ganttproject/src/main/java/net/sourceforge/ganttproject/chart/ChartModelBase.java

```diff
@@ -213,6 +218,21 @@ public DateFormat getShortDateFormat() {
       public String i18n(String key) {
         return GanttLanguage.getInstance().getText(key);
       }
+
+      @Override
+      public Integer getWeekNumber(Date date) {
+        Calendar calendar = CalendarFactory.newCalendar();
+        calendar.setTime(date);
+        Integer weekNum = calendar.get(Calendar.WEEK_OF_YEAR);
+
+        if (!myWeekNumberOption.isChecked()) {
+          return weekNum;
+        }
+
+        calendar.setTime(myTaskManager.getProjectStart());
+        Integer startWeekNum = calendar.get(Calendar.WEEK_OF_YEAR);
+        return weekNum - startWeekNum;
```

Comment:
Our users are not programmers, and most likely start numbering with 1. The first project week should be week no 1